VERSION

```diff
@@ -1 +1 @@
-0.1.4-dev
\ No newline at end of file
+0.2.0-dev
```

Comment:
This is an intentional change, it was originated from a potential breaking change in https://github.com/lacework/terraform-aws-iam-role/pull/12/commits/554dbd506ba918c3b10a14ec9fc8936eabb1455f